tests/foreman/rhci/test_computeresources.py

```diff
@@ -0,0 +1,353 @@
+from robottelo.test import UITestCase
+from robottelo.common import conf
+from robottelo.ui.session import Session
+from robottelo.common.constants import FOREMAN_PROVIDERS
+from robottelo.ui.factory import make_resource
+from ddt import data, ddt
```

Comment:
we are not using data and ddt anymore according #2769. Please use `subTest` instead.